okio-wasifilesystem/src/wasmMain/kotlin/okio/internal/preview1/Preview1.kt

```diff
@@ -130,6 +130,51 @@ internal external fun path_open(
   returnPointer: PointerU8,
 ): Int // should be Short??
 
+/**
+ * path_rename(fd: fd, old_path: string, new_fd: fd, new_path: string) -> Result<(), errno>
+ *
+ * Rename a file or directory.
+ * Note: This is similar to `renameat` in POSIX.
+ */
+@WasmImport("wasi_snapshot_preview1", "path_rename")
+internal external fun path_rename(
+  fd: fd,
+  old_path: PointerU8,
+  old_pathSize: size,
+  new_fd: fd,
+  new_path: PointerU8,
+  new_pathSize: size,
+): Int // should be Short??
```

Comment:
Have a strategy to know about this one?

Reply:
Or you mean it gives you an Int but you think it should actually be a short?

Reply:
I should do an experiment to see if they both work. This maps to errno, which is a 16-bit value, but there’s potentially also minimum sizes for these things? It’s all pretty new.